packages/yoroi-ergo-connector/manifest/manifest.template.js

```diff
@@ -30,7 +30,7 @@ module.exports = ({
     "default_title": displayName,
     default_icon: icons,
   },
-  "content_security_policy": "default-src 'none'; script-src 'self';",
+  "content_security_policy": "default-src 'none'; script-src 'self'; img-src https://* ;",
```

Comment:
If we can't put it in the content script, maybe we can put it inside `inject.js` inside the function `ergo_request_read_access` so that it still executes in the context of the user's page? 🤔